frontend/src/constants/index.ts

```diff
@@ -25,3 +25,116 @@ export const ERROR_MESSAGES = {
   NO_STATION_FOUND: `${ERROR_PREFIX} 해당 충전소가 존재하지 않습니다.`,
   STATION_DETAILS_FETCH_ERROR: `${ERROR_PREFIX} 충전소 세부 정보를 불러올 수 없습니다.`,
 } as const;
+
+export const CHARGER_TYPE = {
+  DC_FAST: 'DC 차데모',
+  AC_SLOW: 'AC 완속',
+  DC_AC_3PHASE: 'DC 차데모+AC3 상',
+  DC_COMBO: 'DC 콤보',
+  DC_DC_COMBO: 'DC 차데모+DC 콤보',
+  DC_AC_DC_COMBO: 'DC 차데모+AC 상+DC 콤보',
+  AC_3PHASE: 'AC3 상',
+} as const;
```

Comment:
```suggestion
export const CHARGER_TYPE = {
  DC_FAST: 'DC 차데모',
  AC_SLOW: 'AC 완속',
  DC_AC_3PHASE: 'DC 차데모+AC 3상',
  DC_COMBO: 'DC 콤보',
  DC_DC_COMBO: 'DC 차데모+DC 콤보',
  DC_AC_DC_COMBO: 'DC 차데모+AC 3상+DC 콤보',
  AC_3PHASE: 'AC 3상',
} as const;
```
그냥 심심해서 구경했는데 3상은 띄우지 않습니다~~

Reply:
중요중요

Reply:
수정 완료했습니다~